src/java.base/share/classes/java/lang/Float16.java

```diff
@@ -649,6 +650,7 @@ public int intValue() {
      * @jls 5.1.3 Narrowing Primitive Conversion
      */
     @Override
+    @IntrinsicCandidate
```

Comment:
Should this be handled trough Idealization xform ?

ConvHF2F + ConvF2L => ConvHF2L